.github/workflows/uberjar.yml

```diff
@@ -56,9 +56,37 @@ jobs:
         path: ~/.m2
         key: ${{ runner.os }}-m2-${{ hashFiles('**/project.clj') }}-${{ hashFiles('**/deps.edn') }}
 
+    - name: Grab previously built front-end (if equivalent)
+      if: "!contains(github.event.head_commit.message, '[ci nocache]')"
+      id: frontend-cache
+      uses: actions/cache@v2
+      with:
+        path: ./resources/frontend_client/
+        key: buildcache-${{ matrix.edition }}-frontend-${{ hashFiles('**/*.js', '**/*.jsx', '**/*.cljc', '**/*.cljs', '**/*.json', '**/yarn.lock', '**/sample-dataset.db.mv.db', '!**/frontend/test/') }}
```

Comment:
This is better than nothing but still not nearly as granular as we might want it.
Do you want me to keep digging deeper for some other paths that might (not) be needed here or do you think this is good enough for now?